apps/passport-client/src/dispatch.ts

```diff
@@ -942,3 +956,67 @@ async function promptToAgreePrivacyNotice(state: AppState, update: ZuUpdate) {
     });
   }
 }
+
+/**
+ * Merge in a PCD collection from an import of backed-up data.
+ */
+async function mergeImport(
+  state: AppState,
+  update: ZuUpdate,
+  collection: PCDCollection,
+  pcdsToMergeIds: Set<PCD["id"]>
+) {
+  const userHasExistingSemaphoreIdentityPCD =
```

Comment:
It would be nice if this filtering logic (special-case for identity, email, and the decision never to replace the same ID) were in a helper function defined once rather than duplicated both here and in the UI.